web-app/src/components/HolidayModal/index.js

```diff
@@ -61,23 +72,45 @@ const HolidayModal = ({
           </Stat>
         </StatWrap>
         {shouldShowAdminControls() && (
+          toggled === false ? <Stat>
+            <ButtonWrap>
+              <Button label="Approve" onClick={approveHoliday} />
+              <Button label="Reject" onClick={ expandRejectHolidayExplanationText } />
+            </ButtonWrap>
+          </Stat> : null
+        )}
+        { toggled && ( <div><StatWrap>
```

Comment:
lots of inconsistency with spaces between brackets etc. would need to get esLint set up in IDE asap